test/Autofac.Test/Features/ResolveAnything/ResolveAnythingTests.cs

```diff
@@ -257,6 +256,16 @@ private static IContainer CreateResolveAnythingContainer()
             return cb.Build();
         }
 
+        public class GenericType<T>
```

Comment:
Where is this getting used in the tests?

Reply:
It's not, my bad; left over from when I looked at bringing in the tests from the original PR. Removed.